warehouse/dbt_project.yml

```diff
@@ -48,6 +48,14 @@ models:
       +materialized: table
       schema: views
 
+    payments_views_staging:
+      +materialized: view
+      schema: payments
+
+    payments_views:
+      +materialized: table
+      schema: views
```

Comment:
nit: do you mind adding TODO comments that `payments_views_staging` should probably end up in `staging` and `payments_views` should end up in `mart_payments`?